### PR TITLE
(v1.0) Re-organized ExcelBuilderAction.php.twig

### PR DESCRIPTION
**Pull request by Restless-ET**
Added  a `getValueForCell` protected function so that extending the values format for the cells will be easier.

For instance if we need to format the value of a field which we know it's a currency, we can easily extends this method and add those special cases. E.g.:

``` php
protected function getValueForCell($field, $rowData)
{
    switch($field) {
        case 'price':
        case 'productionCost':
            $getter = 'get'.ucfirst($field);
            $data = number_format($rowData->$getter(), 2, ',', '.');
            break;
        default:
            $data = parent::getValueForCell($field, $rowData);
            break;
    }
    return $data;
}
```

I don't know if this is the best example, but this can also be used to render boolean field with some text other than `TRUE/FALSE`.

### CODE REVIEW

**Comment by Restless-ET** [comment]
@loostro What's the status on this PR? Are there any further changes/adjustments necessary?

I already have branches on my fork where I've applied this against `v1.1` and `master`. Once you give me your OK for this I can submit the PRs for the other two as well. :-)

File: Resources/templates/CommonAdmin/ExcelAction/ExcelBuilderAction.php.twig [hunk 1]
```diff
@@ -69,35 +69,56 @@ class ExcelController extends ListController
...
-          $getter = '{{ builder_column.getter }}';
-          $data = $rowData;
-          while (($pos = strpos($getter, '.')) > 0){
-            $tempGetter = 'get' . ucfirst(substr($getter, 0, $pos));
+    protected function getValueForCell($field, $rowData)
+    {
+        $getter = $field;
+        $data = $rowData;
+        // Retrieve relations
+        while (($pos = strpos($getter, '.')) > 0) {
```
Comment (sescandell) [inline Resources/templates/CommonAdmin/ExcelAction/ExcelBuilderAction.php.twig]: ping @Restless-ET

**Comment by loostro** [comment]
@Restless-ET I've pushed some PRs to your fork (use PropertyAccesor as @sescandell suggested). Once you merge them, this PR should be auto updated and ready to merge into master.

Also please make a PR into v1.1 branch.

**Comment by bobvandevijver** [comment]
This bundle is no longer maintained. Please use the new https://github.com/symfony2admingenerator/GeneratorBundle.

**Pull request closed without merging**